### DISCUSSION TITLE
URL scheme

### DISCUSSION THREAD

**Discussion by murillodutt** [Goodwy/PlayBooks#46] [General]
Would it be possible to use a URL scheme to download the audiobook to the Playbook directly from a URL? 

On iOS I use BookPlayer APP which allows you to implement a URL, see example:

bookplayer://download?url="https://****.m4b"

[WIKI BookPlayer URL Scheme](https://github.com/TortugaPower/BookPlayer/wiki/URL-scheme)

**Comment by Goodwy** [comment]
This scheme is not supported.

**Reply by murillodutt** [reply]
Implementing a URL scheme for audiobook downloads directly from a URL enhances user convenience, streamlines content acquisition, and potentially attracts new users. This user-centered feature aligns with industry standards, offering a competitive edge. The implementation is technically feasible, requiring minimal development effort.
